src/cmds/Execute.my

```diff
@@ -0,0 +1,15 @@
+package embox.cmd
+
+@AutoCmd
+@Cmd(name = "load_app",
+	help = "Load and execute program",
+	man = '''
+		AUTHORS
+			Artem Sharganov
+	''')
+module load_app {
```

Comment:
Loadable apps are interesting (IMO), so it would be great to see them in some templates. x86/user_apps looks as a good choice. But it be x86/qemu and x86/test/lang too
@anton-bondarev, any thoughts?